src/adapters/adapter.js

```diff
@@ -60,6 +60,35 @@ class Adapter {
             item.text = name
             item.icon = type // uses `type` as class name for tree node
 
+            if (item.patch) {
+               let patch_html = ''
+
+               if (item.patch.action) {
+                 if (item.patch.action === 'add') {
+                   patch_html += '<span class="text-green">added</span>'
+                 }
+                 else if (item.patch.action === 'rename') {
+                   patch_html +=
+                   `<span class="text-green"
+                     title="${item.patch.previous}">renamed
+                    </span>`
+                 }
+               }
+
+               if (item.patch.changes) {
+                 patch_html += `<span>${item.patch.changes} changes</span>`
```

Comment:
I would consider changing the text to `# files` instead of `# changes` -- it's a bit misleading as-is, as I would expect changes to total the number of lines that were modified.